tools/BuildAssets/BuildTasks/Microsoft.Azure.Sdk.Build.Tasks/buildTools.props

```diff
@@ -0,0 +1,11 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+	<PropertyGroup>
```

Comment:
nit: Most msbuild xml files use 2 space indention without tabs.

Reply:
@weshaggard I need to fix my indentation just for xml files in all the various editors I use. I have not been able to get set those defaults just for xml across VS, VSCode and notepad++
I will fix this very soon.